tests/study/test_study_resources.py

```diff
@@ -0,0 +1,129 @@
+import json
+
+from flask import url_for
+
+from dataservice.extensions import db
+from dataservice.api.study.models import Study
+from tests.utils import FlaskTestCase
+
+STUDY_URL = 'api.studies'
+STUDY_LIST_URL = 'api.studies_list'
+
+
+class StudyTest(FlaskTestCase):
+    '''
+    Test study api endopoints
+    '''
+
+    def test_post_study(self):
+        '''
+        Test creating a new study
+        '''
+        response = self._make_study(external_id='TEST')
+        resp = json.loads(response.data.decode('utf-8'))
+
+        self.assertEqual(response.status_code, 201)
+        self._test_response_content(resp, 201)
+
+        self.assertIn('study', resp['_status']['message'])
+        self.assertIn('created', resp['_status']['message'])
+        self.assertNotIn('_id', resp['results'])
+
+        s = Study.query.first()
+        study = resp['results']
+        self.assertEqual(s.kf_id, study['kf_id'])
+        self.assertEqual(s.external_id, study['external_id'])
+
+    def test_get_study(self):
+        '''
+        Test retrieving a study by id
+        '''
+        resp = self._make_study('TEST')
+        resp = json.loads(resp.data.decode('utf-8'))
+        kf_id = resp['results']['kf_id']
+
+        response = self.client.get(url_for(STUDY_URL,
+                                           kf_id=kf_id),
+                                   headers=self._api_headers())
+        resp = json.loads(response.data.decode('utf-8'))
+        self.assertEqual(response.status_code, 200)
+        self._test_response_content(resp, 200)
+
+        study = resp['results']
+        self.assertEqual(kf_id, study['kf_id'])
+
+    def test_patch_study(self):
```

Comment:
Probably a good to check the database as well. Make sure that only the fields in the body were modified and others remained the same.